tests/Registration/RegistrationServiceTest.php

```diff
@@ -55,20 +57,54 @@ public function testRegisterMissingParameters(): void
 
     public function testRegisterCreate(): void
```

Comment:
I agree with GitHub; it will be good to cover mutation cases

Reply:
Fixed